### ISSUE TITLE
[Bug] update_infos_to_v2 fails for ScanNet test split

### ISSUE THREAD

**Issue by filaPro** [open-mmlab/mmdetection3d#2268]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection3d/issues) and [Discussions](https://github.com/open-mmlab/mmdetection3d/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://github.com/open-mmlab/mmdetection3d/blob/1.1/docs/en/notes/faq.md) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (dev)](https://github.com/open-mmlab/mmdetection3d/tree/dev) or [latest version (1.x)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

1.1x branch https://github.com/open-mmlab/mmdetection3d/tree/1.1

### Environment

not important

### Reproduces the problem - code sample

-

### Reproduces the problem - command or script

```shell
python tools/dataset_converters/update_infos_to_v2.py --dataset scannet --pkl-path data/scannet/scannet_tr3d_infos_test.pkl --out-dir work_dirs/tmp/
```

### Reproduces the problem - error message

```
Traceback (most recent call last):
  File "/opt/project/tools/dataset_converters/update_infos_to_v2.py", line 1142, in <module>
    update_pkl_infos(
  File "/opt/project/tools/dataset_converters/update_infos_to_v2.py", line 1125, in update_pkl_infos
    update_scannet_infos(pkl_path=pkl_path, out_dir=out_dir)
  File "/opt/project/tools/dataset_converters/update_infos_to_v2.py", line 615, in update_scannet_infos
    ori_info_dict['pts_semantic_mask_path']).name
KeyError: 'pts_semantic_mask_path'
```

### Additional information

ScanNet test annotations don't have 'pts_semantic_mask_path' and 'pts_instance_mask_path'.

**Comment by filaPro** [comment]
```
diff --git a/tools/dataset_converters/update_infos_to_v2.py b/tools/dataset_converters/update_infos_to_v2.py
index 0af78609..ad4a99a2 100644
--- a/tools/dataset_converters/update_infos_to_v2.py
+++ b/tools/dataset_converters/update_infos_to_v2.py
@@ -611,19 +611,25 @@ def update_scannet_infos(pkl_path, out_dir):
             'point_cloud']['num_features']
         temp_data_info['lidar_points']['lidar_path'] = Path(
             ori_info_dict['pts_path']).name
-        temp_data_info['pts_semantic_mask_path'] = Path(
-            ori_info_dict['pts_semantic_mask_path']).name
-        temp_data_info['pts_instance_mask_path'] = Path(
-            ori_info_dict['pts_instance_mask_path']).name
+        if 'pts_semantic_mask_path' in ori_info_dict:
+            temp_data_info['pts_semantic_mask_path'] = Path(
+                ori_info_dict['pts_semantic_mask_path']).name
+        if 'pts_instance_mask_path' in ori_info_dict:
+            temp_data_info['pts_instance_mask_path'] = Path(
+                ori_info_dict['pts_instance_mask_path']).name
 
         # TODO support camera
         # np.linalg.inv(info['axis_align_matrix'] @ extrinsic): depth2cam
-        anns = ori_info_dict['annos']
-        temp_data_info['axis_align_matrix'] = anns['axis_align_matrix'].tolist(
-        )
-        if anns['gt_num'] == 0:
-            instance_list = []
+        if 'annos' in ori_info_dict:
+            anns = ori_info_dict['annos']
+            temp_data_info['axis_align_matrix'] = \
+                anns['axis_align_matrix'].tolist()
         else:
+            anns = dict(gt_num=0)
+
+        instance_list = []
+        ignore_class_name = set()
+        if anns['gt_num'] > 0:
             num_instances = len(anns['name'])
             ignore_class_name = set()
             instance_list = []

```

**Comment by ZwwWayne** [comment]
Hi @filaPro ,
Thanks for your kind PR, @Xiangxu-0103 will fix this issue soon.

**Comment by Xiangxu-0103** [comment]
Thanks for your kind reminder. It has been fixed in #2285.
